### ISSUE TITLE
cc-block-details: missing focus outline style for the Command line button

### ISSUE THREAD

**Issue by florian-sanders-cc** [CleverCloud/clever-components#1547]
## Context

The `Command line` button has no focus styling:

<img width="203" height="87" alt="Black outline surrouding the Command line button" src="https://github.com/user-attachments/assets/104f6e01-8b60-4038-8779-7f94afd43a00" />

We need to add `:focus-visible` & use the `--cc-focus-outline` & `--cc-focus-outline-offset`.
